salt/config/__init__.py

```diff
@@ -60,8 +60,8 @@
     _DFLT_MULTIPROCESSING_MODE = True
 
 FLO_DIR = os.path.join(
-        os.path.dirname(__file__),
-        'daemons', 'flo')
+            os.path.dirname(os.path.dirname(__file__)),
```

Comment:
If this is relative to the salt source root directory we can use `salt.syspaths.INSTALL_DIR` to get the source root directory...